src/main/java/io/github/jhipster/config/apidoc/SwaggerConfiguration.java

```diff
@@ -19,42 +19,34 @@
 
 package io.github.jhipster.config.apidoc;
 
-import static io.github.jhipster.config.JHipsterConstants.SPRING_PROFILE_SWAGGER;
-import static springfox.documentation.builders.PathSelectors.regex;
-
-import java.nio.ByteBuffer;
-import java.util.*;
-
+import com.fasterxml.classmate.TypeResolver;
+import io.github.jhipster.config.JHipsterProperties;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
-import org.springframework.context.annotation.*;
+import org.springframework.context.annotation.Bean;
 import org.springframework.http.ResponseEntity;
 import org.springframework.util.StopWatch;
-
-import com.fasterxml.classmate.TypeResolver;
-
-import io.github.jhipster.config.JHipsterProperties;
-import springfox.bean.validators.configuration.BeanValidatorPluginsConfiguration;
 import springfox.documentation.schema.TypeNameExtractor;
 import springfox.documentation.service.ApiInfo;
 import springfox.documentation.service.Contact;
 import springfox.documentation.spi.DocumentationType;
 import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+
+import static springfox.documentation.builders.PathSelectors.regex;
 
 /**
  * Springfox Swagger configuration.
  *
  * Warning! When having a lot of REST endpoints, Springfox can become a performance issue. In that case, you can use a
  * specific Spring profile for this class, so that only front-end developers have access to the Swagger view.
  */
-@Configuration
-@ConditionalOnClass({ ApiInfo.class, BeanValidatorPluginsConfiguration.class })
-@EnableSwagger2
-@Import(BeanValidatorPluginsConfiguration.class)
-@Profile(SPRING_PROFILE_SWAGGER)
+@Deprecated
```

Comment:
Can this be removed in favour if new config instead of deprecation?

Reply:
Fixed